website/src/pages/BlogPage/index.tsx

```diff
@@ -189,7 +190,7 @@ const BlogPage: React.FC = () => {
                   {socialLinks.map(({ label, imgURL }) => {
                       return (   
                         (label === SOCIAL_PLATFORMS.SLACK) ? 
-                          <div className={["addthis_inline_share_toolbox", classes.socialIconButton].join(' ')} key={label}></div>
+                          <SlackIcon key={label} />
```

Comment:
can we name this as `SlackShareIcon`? Because we have another slack icon in the footer for joining slack community. I think we should differentiate this based on the functionality